firestore.rules

```diff
@@ -1,8 +1,25 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /{document=**} {
-      allow read, write: if true;
-    }
+        match /items/{itemId} {
+          allow create: if request.auth != null;
+          allow update, delete: if request.auth.uid == request.resource.data.userId;
+          allow read: if true;
+        }
+        match /users/{userId} {
+          allow create, read: if true;
+          allow update: if request.auth != null && request.auth.uid == userId;
```

Comment:
```suggestion
          allow read: if true;
          allow create, update: if request.auth != null && request.auth.uid == userId;
```

Reply:
Mh this can't work, we need to be logged out to create a user

Reply:
When the user logs in with Google, the `auth` value is set and after that we add it to Firestore so it may work. Could you try it please?
https://github.com/Sharingang/Sharingang-Android/blob/5dcab0936429b0b248da78fdb5233a5532a22de5/app/src/main/java/com/example/sharingang/auth/AuthHelper.kt#L38-L42

Reply:
You're right, it works